LiskHQ/lisk
# Require db repositories statically - Closes #1541

### What was the problem?

Adding files to the `db/repos` directory which are not valid database repository modules caused an unexpected error when starting the application.

### How did I fix it?

By requiring db repositories statically using an index file.

Replaces #1542 

### How to test it?

Add a module not defined by `db/repos/index.js` to the `db/repos` directory and the start the application. The application should start without any fatal errors.

### Review checklist

* The PR solves #1541 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
